.remarkignore

```diff
@@ -0,0 +1 @@
+/docs/blog/
```

Comment:
> We also disable docs/blog for now, there are a lot of old blog posts that don't follow style guides.

Can those be auto-fixed or do they require manual fixes?

Reply:
Auto-fixing currently has a lot of styling inconsistencies with Prettier that I don't know how to resolve (which I've added to the main issue). There are also a lot of rules that don't have quick fixes, such as `no-heading-punctuation` and `no-emphasis-as-heading`. My reasoning was that it's better to have the linter, even if we don't have the formatter to auto-fix things, so it can deal with the errors that prettier can't (especially ones that can't be auto-fixed).